bundles/config/org.eclipse.smarthome.config.core/src/main/java/org/eclipse/smarthome/config/core/MultipleInstanceServiceInfo.java

```diff
@@ -0,0 +1,55 @@
+/**
+ * Copyright (c) 2014,2018 Contributors to the Eclipse Foundation
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information regarding copyright ownership.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.eclipse.smarthome.config.core;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+
+/**
+ * Interface marking the service with service.pid as one that can be instantiated multiple times with different
+ * configurations
+ *
+ * @author Stefan Triller - initial contribution
+ *
+ */
+@NonNullByDefault
+public interface MultipleInstanceServiceInfo {
```

Comment:
I understand that you need a "dummy" or "marker" instance of a service in order to let the framework know that there _is_ a type of service which can be configured multiple times.

For "singleton" services we are using service properties for the "meta-data" which need to be returned by the below methods. I'm not saying that it is the _better_ method of providing the meta-data, it's just _different_. And IMHO this difference is confusing. Can't they also be exposed as properties of such a dummy/marker service instance? Also, why does it need to implement this interface? Shouldn't a property suffice for that purpose?

Reply:
Alright. I agree that its better to have it consistent with the singleton services. So I dropped the new Interface and use the properties on the marker service now.